core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/ui/dot-experiments-inline-edit-text/dot-experiments-inline-edit-text.component.html

```diff
@@ -0,0 +1,55 @@
+<p-inplace [disabled]="disabled">
+    <ng-template pTemplate="display">
+        <div class="flex">
+            <span *ngIf="text; else noTextDefinedTpl" data-testId="text-input">
+                {{ text }}
+            </span>
+            <i
+                class="pi pi-pencil cursor-pointer"
+                *ngIf="!disabled"
+                data-testId="text-input-icon"
+            ></i>
+        </div>
+
+        <ng-template #noTextDefinedTpl>
+            <span class="text-muted" data-testId="empty-text-message">{{
+                emptyTextMessage | dm
+            }}</span>
+        </ng-template>
+    </ng-template>
+
+    <ng-template pTemplate="content">
+        <form [formGroup]="form" novalidate>
+            <div class="flex gap-2 control">
+                <div class="p-input-icon-right flex flex-column w-full">
+                    <input
+                        [maxLength]="maxCharacterLength + 1"
```

Comment:
Why do we add `1` to the `maxCharacterLength`?

Reply:
I removed that.